R/slice.R

```diff
@@ -356,7 +353,27 @@ check_constant <- function(x, name, error_call = caller_env()) {
   })
 }
 
-check_slice_size <- function(n, prop, error_call = caller_env()) {
+check_slice_dots <- function(..., n, prop, error_call = caller_env()) {
+  # special case to capture e.g. slice_head(2)
+  if (missing(n) && missing(prop)) {
+    # capture as quosure so that we can label
+    dots <- enquos(...)
+
+    if (length(dots) == 1L && names2(dots)[1] == "") {
+      slice_call <- error_call$.Generic
+      bullets <- c(
+        "`n` must be explicitly named.",
+        i = glue("Did you mean `{slice_call}(n = {as_label(dots[[1]])})` ?")
```

Comment:
```suggestion
        i = glue("Did you mean `{slice_call}(n = {as_label(dots[[1]])})`?")
```

Do we typically add a space between expressions and punctuation?

Reply:
Thanks.